base-theme/layouts/partials/resource_url.html

```diff
@@ -9,8 +9,10 @@
 */}}
 {{- $prefix := getenv "RESOURCE_BASE_URL" | default "" -}}
 {{- $url := printf "%s%s" (strings.TrimSuffix "/" $prefix) (urls.Parse $url).Path -}}
-{{- if hasPrefix $url "http" -}}
-  {{- return $url -}}
-{{- else -}}
+{{- if not (hasPrefix $url "http") -}}
   {{- partial "site_root_url.html" $url -}}
```

Comment:
You are correct, I'm not sure what I was thinking here. To be honest, I'm not sure we need to reference `site_root_url` here. This partial was created as a way to bypass the `baseURL` setting and reference the root of the site directly, for links in the header that need to do that such as `/about`, etc. If you look at that partial, you'll see it uses a bunch of `printf` statements to print URLs directly, and this is because it was written near the start of the project and Hugo did not support calling `return` in partials back then. We should revisit how this partial works, or if it is even needed but that is work for another PR.